docs/install-upgrade/install.md

```diff
@@ -84,7 +94,36 @@ This control node should be given a static IP address. Either a lease or statica
 
 ### Configure Management Network
 
-The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node will run a DHCP and small http servers. The management network is not accessible to machines or devices not associated with the fabric.
+The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node via their building IT network. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node will run a DHCP and small http servers. The management network is not accessible to machines or devices not associated with the fabric. The following diagram illustrates 3 networks for the control node. A "Local IT"  network that is used for command and control or IPMI. The other network is exclusive to Hedgehog, the best solution is a dedicated network switch, but a private exclusive vlan is potentially workable.
```

Comment:
Side note: Would be easier to review the changes on lines like this if you wrapped the lines. I prefer wrapping text for docs; your preferences might differ and that's OK, I don't think we have a convention for that.

Reply:
I have been experimenting with my editor to make this easier. What sent me down the path of one big line was the drudgery of having to "re-flow" text if I was editing and the location of the line breaks changed.

Reply:
Maybe your editor has a feature to reflow text automatically, or at least with a simple command? Such as `gqap` for the whole paragraph in Vim.

Reflowing is not the perfect solution though, I don't like that it messes up the diff if you want to reflow after a small change in the paragraph. Overall, I still find it easier to work with shorter lines :shrug: